src/app/shared/components/button/button.component.html

```diff
@@ -0,0 +1,22 @@
+<section>
+  <div class="example-button-row">
+    <button *ngIf="element?.type === 'basic' || buttonType === 'basic'"
```

Comment:
create ENUM if possible
```suggestion
    <button *ngIf="element?.type === ENUM  || buttonType === ENUM"
```